hugegraph-core/src/main/java/com/baidu/hugegraph/schema/SchemaLabel.java

```diff
@@ -29,13 +29,13 @@
 import com.baidu.hugegraph.backend.id.Id;
 import com.baidu.hugegraph.backend.id.IdGenerator;
 import com.baidu.hugegraph.type.HugeType;
-import com.baidu.hugegraph.type.Indexfiable;
+import com.baidu.hugegraph.type.Indexifiable;
 import com.baidu.hugegraph.type.Propfiable;
 import com.baidu.hugegraph.util.E;
 import com.google.common.base.Objects;
 
 public abstract class SchemaLabel extends SchemaElement
-                                  implements Indexfiable, Propfiable {
+                                  implements Indexifiable, Propfiable {
```

Comment:
could you also rename Propfiable to Propertiable?

In fact all classes of `com.baidu.hugegraph.type.*fiable` can be reconsidered

Reply:
I've renamed to Propertiable and selected Typeable.

Reply:
`Namifiable` should be `Nameable`...